metricbeat/module/kubernetes/apiserver/_meta/fields.yml

```diff
@@ -4,26 +4,129 @@
     Kubernetes API server metrics
   release: ga
   fields:
-    - name: request.client
+    - name: client
       type: keyword
       description: >
         Client doing the requests
-    - name: request.resource
+    - name: resource
       type: keyword
       description: >
         Requested resource
-    - name: request.subresource
+    - name: subresource
       type: keyword
       description: >
         Requested subresource
-    - name: request.scope
+    - name: scope
       type: keyword
       description: >
         Request scope (cluster, namespace, resource)
-    - name: request.verb
```

Comment:
These look like breaking changes, could we do this keeping compatibility? If not, we'd have to add a breaking change entry to the changelog.

Reply:
true, will revert back to previous names. Thanks for pointing out